guide/14-deep-learning/labeling-text-using-doccano.ipynb

```diff
@@ -0,0 +1,136 @@
+{
```

Comment:
<p>American English uses labeling instead of labelling. However, the title uses Labelling. Spell it as labeling</p><p>Change "We will use Doccano to create labeled data for training our&nbsp;<code>EntityRecongnizer()</code>&nbsp;from&nbsp;<code>arcgis.learn</code>" to "Doccano can be used to create labeled data for training &nbsp;the <code>EntityRecongnizer</code>&nbsp;model in <code>arcgis.learn</code> .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/644/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>